docs/pages/database-access/guides/aws-cassandra-keyspaces.mdx

```diff
@@ -3,11 +3,11 @@ title: Database Access with Amazon Keyspaces (Apache Cassandra)
 description: How to configure Teleport database access with Amazon Keyspaces (Apache Cassandra)
 ---
 
-(!docs/pages/includes/database-access/db-introduction.mdx  dbType="Amazon Keyspaces (Apache Cassandra)" dbConfigure="Amazon Keyspaces database with IAM authentication" dbName="Amazon Keyspaces" !)
+(!docs/pages/includes/database-access/db-introduction.mdx dbType="AWS Keyspaces (Apache Cassandra)" dbConfigure="with IAM authentication"!)
```

Comment:
Several lint including this one is correct on product name. It should be Amazon Keyspaces, Amazon ElastiCache, etc.